tools-v2/pkg/cli/command/curvebs/status/client/client.go

```diff
@@ -92,9 +94,7 @@ func (cCmd *ClientCommand) Init(cmd *cobra.Command, args []string) error {
 	}
 
 	if len((*results).([]map[string]string)) == 0 {
-		retErr := cmderror.ErrBsGetClientStatus()
```

Comment:
delete this error number in `tools-v2/internal/error/error.go`?

Reply:
The err code is used by at func GetClientStatus.